src/Components/Calculator.js

```diff
@@ -1,43 +1,68 @@
 import React from 'react';
 import '../index.css';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust  and clean code.

Reply:
Thank you for the notice. @rloterh